app/components/search/SearchMap.jsx

```diff
@@ -55,7 +55,15 @@ const SearchMap = ({ hits, userLocation }) => {
     return null;
   }
 
-  const markers = hits.map((hit, index) => <CustomMarker lat={hit._geoloc ? hit._geoloc.lat : 0} lng={hit._geoloc ? hit._geoloc.lng : 0} key={hit.objectID} index={index} />);
+  const markers = hits.map(function(hit, index) {
```

Comment:
In terms of code style, we prefer using arrow functions in many cases, but especially when passing anonymous functions to functional programming functions like `map()`. If you want to break it across multiple lines, you can put parentheses around the arrow function body, like so:

```js
  const markers = hits.map((hit, index) => (
    <CustomMarker
      lat={hit._geoloc ? hit._geoloc.lat : 0}
      lng={hit._geoloc ? hit._geoloc.lng : 0}
      key={hit.objectID}
      index={index} />
  ));
```

Reply:
I'm going to go ahead and make the minor formatting changes here and push it up to this branch, and then I can merge when Travis completes.